README.md

```diff
@@ -88,19 +88,35 @@ Installation can be done in one of 2 options:
 
 
 ## Changelog
+### v0.9.3
+- tbtadm: added `peers` command and xdomain is now shown in the topology output
```

Comment:
What about the addition of `tbtxdomain` for auto-loading `tuhnderbolt-net` driver?

Reply:
Added line to changelog about autoload